src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/betingelser/aktkrav/SjekkOmMorOppgittUføre.java

```diff
@@ -20,6 +20,9 @@
 
     @Override
     public Evaluation evaluate(FastsettePeriodeGrunnlag fastsettePeriodeGrunnlag) {
+        if (fastsettePeriodeGrunnlag.isBareFarHarRettMorUføretrygd()) {
```

Comment:
Føler at denne sjekken ser på mer enn bare om det er oppgitt uføre
"Har søker oppgitt at mor mottar uføretrygd ?"

Reply:
Den dekker egentlig både avslag-akt-12 og avslag-akt-11 i flyten - men siden vi på sikt går over fra uføre på perioder til oppgitt rettighet så bør disse antagelig slås sammen

Reply:
Har utvidet RettOgOmsorg med felt for oppgittUføre (vs bekreftet som allerede finnes)